M2/Macaulay2/m2/document.m2

```diff
@@ -420,8 +420,10 @@ hasDocumentation = key -> null =!= fetchAnyRawDocumentation makeDocumentTag(key,
 locate DocumentTag := tag -> new FilePosition from (
     if (rawdoc := fetchAnyRawDocumentation tag) =!= null
     then (
-	minimizeFilename((package rawdoc.DocumentTag)#"source directory" |
-	    rawdoc#"filename"),
+	fn := rawdoc#"filename";
+	if isMember(fn, {"stdio", "currentString"}) then fn
+	else minimizeFilename(
+	    (package rawdoc.DocumentTag)#"source directory" | fn),
```

Comment:
This feels like a hack to fix another hack (which is calling `document` directly instead of loading from a file). Maybe instead we should change `source directory` of `User` to be empty string?